### PR TITLE
create metaos page on the what we do

### PR DESCRIPTION
**Pull request by peth-yursick**

### CODE REVIEW

**Review by luxumbra — CHANGES_REQUESTED** [review]
Nice read but there's no sidebars.js update, so when you view the page it has no sidebar. Also, the Frontmatter *must* be added on the *first* line of the file, otherwise it wont treat it as Frontmatter and will render the contents to the page. 

image

File: docs/what-we-do/metaos.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+
```
Comment (luxumbra) [inline docs/what-we-do/metaos.md:1]: This line needs to be removed to make the `title` in the Frontmatter render as it should.

**Comment by luxumbra** [comment]
If you don't fancy doing the fixes on this @peth-yursick, let us know.

**Comment by peth-yursick** [comment]
Thank you. The sidebar update comes in #320, I didn't want to create another PR just for the sidebar update & I knew the desktop app would be annoying 😅

**Review by luxumbra — APPROVED** [review]
LGTM.

**Pull request merged**